code/datums/gamemode/factions/syndicate/nukeops.dm

```diff
@@ -26,6 +26,83 @@
 		. += "<BR>The operatives bought:<BR>"
 		for(var/entry in faction_scoreboard_data)
 			. += "[entry]<BR>"
+	var/diskdat = null
+	var/bombdat = null
+	var/opkilled = 0
+	var/oparrested = 0
+	var/alloparrested = 0
+	//var/nukedpenalty = 1000
+	for(var/datum/role/R in members)
+		var/datum/mind/M = R.antag
+		if(!M || !M.current)
+			opkilled++
+			continue
+		var/turf/T = M.current.loc
+		if(T && istype(T.loc, /area/security/brig))
+			oparrested++
+		else if(M.current.stat == DEAD)
+			opkilled++
+	if(peak_member_amount == oparrested)
+		alloparrested = 1
+		score.crewscore += oparrested * 2000
+	score.crewscore += opkilled * 250
+	score.crewscore += oparrested * 1000
+	//if(score.scores["nuked"])
+		//score.crewscore -= nukedpenalty
+
+	if(nukedisk)
+		var/atom/disk_loc = nukedisk.loc
+		while(!istype(disk_loc, /turf))
+			if(istype(disk_loc, /mob))
+				var/mob/M = disk_loc
+				diskdat += "Carried by [M.real_name] "
+			if(istype(disk_loc, /obj))
+				var/obj/O = disk_loc
+				diskdat += "in \a [O.name] "
+			disk_loc = disk_loc.loc
+		diskdat += "in [disk_loc.loc]"
+		/*score.scores["disc"] = 1
+		if(istype(disk_loc,/mob/living/carbon))
+			var/area/bad_zone1 = locate(/area) in areas
+			if(location in bad_zone1)
+				score.scores["disc"] = 0
+			if(istype(get_area(nukedisc),/area/syndicate_mothership))
+				score.scores["disc"] = 0
+			if(istype(get_area(nukedisc),/area/wizard_station))
+				score.scores["disc"] = 0
+			if(nukedisk.loc.z != map.zMainStation)
+				score.scores["disc"] = 0*/
+
+	bombdat = nuked_area
+	if(!nuked_area)
+		for(var/obj/machinery/nuclearbomb/nuke in nuclear_bombs)
+			if(nuke.r_code == "LOLNO")
+				continue
+			bombdat = get_area(nuke)
+			/*nukedpenalty = 50000 //Congratulations, your score was nuked
+			var/turf/T = get_turf(nuke)
+			if(istype(T,/area/syndicate_mothership) || istype(T,/area/wizard_station) || istype(T,/area/solar/) || istype(T,/area))
+				nukedpenalty = 1000
+			else if (istype(T,/area/security/main) || istype(T,/area/security/brig) || istype(T,/area/security/armory) || istype(T,/area/security/checkpoint2))
+				nukedpenalty = 50000
+			else if (istype(T,/area/engine))
+				nukedpenalty = 100000
+			else
+				nukedpenalty = 5000*/
+			//break
+	if(!diskdat)
+		diskdat = "Uh oh. Something has fucked up! Report this."
```

Comment:
I'd recommend not including a Something has fucked up! message publically. Log it with log_admins or similar (include useful variable info if you can), and have the location info read out on the board as "Unknown". After all, this can happen in the rare cases of disks/nukes getting deleted as funny admeme jokes.

Even if you leave the message unchanged, try to include the extra logging for post-round analysis of what could have happened... like, do the disk/nuke even exist? If you don't want to do this, just mark it resolved.

Reply:
this was carried over from the old old era of this being commented out after the move away from gamemodes, so i'll just do your suggestions